### PR TITLE
{bio}[system/system] SNPTEST v2.5.6

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-el8.cloud.in.bmrc.ox.ac.uk - Linux Rocky Linux 8.10, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/77fb71693d87d5dfca884893a581012b for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/s/snptest/snptest-2.5.6.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+easyblock = 'Tarball'
+
+name = 'snptest'
```
Comment (branfosj) [inline easybuild/easyconfigs/s/snptest/snptest-2.5.6.eb:3]: The documentation uses `SNPTEST` consistently for the name. Should we also use that?

File: easybuild/easyconfigs/s/snptest/snptest-2.5.6.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ Ability to condition upon an arbitrary set of covariates and/or SNPs.
+ Various different methods for the dealing with imputed SNPs.
+"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://www.well.ox.ac.uk/~gav/resources/']
+sources = ['snptest_v2.5.6_CentOS_Linux7.8-x86_64_dynamic.tgz']
+checksums = ['c2c829def961dd2f6377c388d8aa22cab17945961c47e39c4a94493466c0a52e']
+
```
Comment (branfosj) [inline easybuild/easyconfigs/s/snptest/snptest-2.5.6.eb:21]: Should we add a postinstall command to symlink `snptest` to `snptest_v2.5.6`?

```
postinstallcmds = ["cd %(installdir)s && ln -s snptest_v2.5.6 snptest"]
```

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/s/snptest/snptest-2.5.6.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/s/snptest/snptest-2.5.6.eb:3]: Yes, you're right. I was going on an old internal module that used lowercase.

File: easybuild/easyconfigs/s/snptest/snptest-2.5.6.eb (hunk 2 above)
Comment (verdurin) [reply easybuild/easyconfigs/s/snptest/snptest-2.5.6.eb:21]: Yes, would be a friendlier approach.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-el8.cloud.in.bmrc.ox.ac.uk - Linux Rocky Linux 8.10, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/7944777dad94d783510b7f59925d1fac for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen3

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/branfosj/59b4a0146787449fa4fb2efa9ac5a4bc for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/d28370e37f0eb8f7388c9d2e204c7368 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0210u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8480CL (icelake), Python 3.6.8
See https://gist.github.com/branfosj/33b607b9cfb1a4f34d077aaac5f8d002 for a full test report.

**Pull request merged**